### PR TITLE
[test] Adapt IOR checks to build systems

### LINKED ISSUE #497: Adapt IOR checks to the new build systems feature

### PR DESCRIPTION
**Pull request by teojgo**
Fixes #497

### CODE REVIEW

File: cscs-checks/system/io/ior_check.py [hunk 1]
```diff
@@ -179,13 +196,15 @@ def __init__(self, name, fs_mount_point, ior_type, num_tasks, **kwargs):
         self.tags = {'monch_acceptance'}
 
 
+@rfm.parameterized_test(*(['/mnt/lnec', 'MPIIO', task]
+                          for task in [40, 80, 160]))
```
Comment (vkarak) [inline cscs-checks/system/io/ior_check.py]: `task` -> `num_tasks` or `nt`.

**Comment by teojgo** [comment]
@jenkins-cscs retry dom

**Comment by teojgo** [comment]
There was no `maintenance` tag, there was the `ops` one.

**Comment by vkarak** [comment]
@teojgo No, there should be a `maintenance` tag:

- On master:

```
./bin/reframe -C config/cscs.py -c cscs-checks/system/io/ior_check.py -t maintenance -l
```

```
List of matched checks
======================
  * ior_read_check_users (found in /users/karakasv/Devel/reframe/cscs-checks/system/io/ior_check.py)
        descr: IOR check (/users)
        tags: {/users,read,ops,maintenance}, maintainers: ['SO', 'MP']
  * ior_write_check_users (found in /users/karakasv/Devel/reframe/cscs-checks/system/io/ior_check.py)
        descr: IOR check (/users)
        tags: {/users,write,ops,maintenance}, maintainers: ['SO', 'MP']
Found 2 check(s).
```

- On your branch

```
List of matched checks
======================
Found 0 check(s).
```

**Pull request merged**